### DISCUSSION TITLE
less strict options for private group chat?

### DISCUSSION THREAD

**Discussion by k0ste** [metalmatze/alertmanager-bot#164] [Help/Support]
Currently, If admin was invited bot to private group chat (standard business practice for support teams in CIS), chat members can't iterate with bot. Can't see current alarms and silences, it's possible to resolve this like "allow commands from group members" or may be from list of usernames?

**Comment by metalmatze** [comment]
Currently not I'm afraid.
I'm reworking things right now though so that this should soon be possible.
First ideas have been written down here https://github.com/metalmatze/alertmanager-bot/pull/157
